.editorconfig

```diff
@@ -5,7 +5,6 @@ charset = utf-8
 end_of_line = lf
 indent_size = 2
 indent_style = space
-insert_final_newline = true
```

Comment:
This was getting annoying when working with sha files, should be covered by pre-commit